URL+Chartboost.swift

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022-2023 Chartboost, Inc.
```

Comment:
Should this file be inside the ReferenceSdk directory, since it's used by the reference SDK?

Reply:
Updated and ready for review now